test_scripts/Polices/External_UCS/008_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct_with_invalid_type_of_params.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_on" struct
+-- with invalid type of params
```

Comment:
@dboltovskyi, as row 4 is part of name of requirement, please put it on row 3.
With search of row 3 in JIRA are listed 2 requirements.